KMB/make_KMB_info.py

```diff
@@ -702,25 +717,31 @@ def municipal_subcategory(self, cat_base, cache):
                     test_cat = None
         return test_cat
 
-    def make_item_class_categories(self, cache):
+    def isolate_primary_class(self):
         """
-        Construct categories from the item class values.
+        Discover which, if any, of the item classes is the primary one.
 
-        :param cache: cache for category existence
+        :return: the matching class
         """
         primary_classes = self.kmb_info.mappings['primary_classes']
-
-        # find the class/tag that is also in primary_classes
-        primary_tag = None
         intersection = list(set(primary_classes) & set(self.item_classes))
```

Comment:
Since we're not using unit tests here, could you give some examples of the two lists?